tests/unit/test_openapi/test_schema.py

```diff
@@ -319,8 +319,8 @@ class MyDataclass:
     assert schema.properties["constrained_int"].exclusive_maximum == 10  # type: ignore
     assert schema.properties["constrained_float"].minimum == 1  # type: ignore
     assert schema.properties["constrained_float"].maximum == 10  # type: ignore
-    assert date.fromtimestamp(schema.properties["constrained_date"].exclusive_minimum) == historical_date  # type: ignore
-    assert date.fromtimestamp(schema.properties["constrained_date"].exclusive_maximum) == today  # type: ignore
+    assert schema.properties["constrained_date"].exclusive_minimum == historical_date  # type: ignore
+    assert schema.properties["constrained_date"].exclusive_maximum == today  # type: ignore
```

Comment:
I don't get how these were passing before, and what we should be testing here.. why did they need to be converted to date before, and why don't they now. How were they even getting set to the right attribute before this PR?